apps/omg/test/omg/state/transaction_test.exs

```diff
@@ -201,7 +199,10 @@ defmodule OMG.State.TransactionTest do
                Transaction.Recovered.recover_from(ExRLP.encode([sigs, @payment_marker, inputs, outputs, ""]))
 
       assert {:error, :malformed_metadata} =
-               Transaction.Recovered.recover_from(ExRLP.encode([sigs, @payment_marker, inputs, outputs, <<0::288>>]))
+               Transaction.Recovered.recover_from(ExRLP.encode([sigs, @payment_marker, inputs, outputs, <<1::224>>]))
+
+      assert {:error, :malformed_metadata} =
+               Transaction.Recovered.recover_from(ExRLP.encode([sigs, @payment_marker, inputs, outputs, <<2::288>>]))
```

Comment:
this is testing module `Transaction.Recovered`, why is this in `transaction_test.exs`?

Reply:
yeah, leftovers. Did a quick split